build.gradle

```diff
@@ -67,8 +67,9 @@ dependencies {
     runtime 'org.apache.logging.log4j:log4j-slf4j-impl:2.8.1'
 
     // Database connectors
-    compile 'com.zaxxer:HikariCP:2.6.3'
-    runtime 'org.mariadb.jdbc:mariadb-java-client:2.0.3'
+    compile 'com.zaxxer:HikariCP:3.2.0'
+    //runtime 'org.mariadb.jdbc:mariadb-java-client:2.0.3'
```

Comment:
1. This removes mariadb support, which would be a breaking change. additionally it's commented.